src/spaces/continuous.jl

```diff
@@ -141,7 +141,7 @@ end
 function move_agent!(agent::A, pos::ValidPos, model::ABM{<:ContinuousSpace{D},A}
     ) where {D, A<:AbstractAgent}
     space_size = spacesize(model)
-    all(i -> 0 <= pos[i] < space_size[i], 1:D) || error("position is outside space extent!")
+    all(i -> 0 <= pos[i] <= space_size[i], 1:D) || error("position is outside space extent!")
```

Comment:
why was equal added here? The extent end is not in the space, because it is no covered by the underlying grid space.

Reply:
since floating point approximation makes it possible to arrive at the extent unfortunately by https://github.com/JuliaDynamics/Agents.jl/issues/853